lib/arbre.rb

```diff
@@ -17,5 +17,5 @@ module Arbre
 require 'arbre/component'
 
 if defined?(Rails)
-  require 'arbre/rails'
+  require 'arbre/railtie'
```

Comment:
This line should be covered but realize the rails test setup is a little awkward. It's trying to only require where needed but the rails tests are run as part of the full test suite so this block won't ever run. I prefer to have coverage here rather than include the railtie file directly.

To do that we'll need to update the both spec helper files with the following changes that pass on all gemfiles.

```diff
diff --git a/spec/spec_helper.rb b/spec/spec_helper.rb
index b49195c..d82bc64 100644
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -12,6 +12,7 @@ if ENV.fetch('COVERAGE', false)
 end
 
 require 'support/bundle'
+require 'combustion'
 require 'arbre'
 
 def arbre(&block)
```

And the following for the `rails_spec_helper.rb` file. The bundler portion can be removed since the `spec_helper.rb` requires a support file that does the same.

```diff
diff --git a/spec/rails/rails_spec_helper.rb b/spec/rails/rails_spec_helper.rb
index d6c69ce..19afc60 100644
--- a/spec/rails/rails_spec_helper.rb
+++ b/spec/rails/rails_spec_helper.rb
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
-require 'rubygems'
-require 'bundler/setup'
-
-require 'combustion'
+require 'spec_helper'
 
 Combustion.path = 'spec/rails/stub_app'
 Combustion.initialize! :action_controller,
@@ -11,14 +8,8 @@ Combustion.initialize! :action_controller,
 require 'rspec/rails'
 require 'capybara/rspec'
 require 'capybara/rails'
-
-require 'spec_helper'
-
 require 'rails/support/mock_person'
 
-# Ensure that the rails plugin is installed
-require 'arbre/rails'
-
 module AdditionalHelpers
 
   def protect_against_forgery?
```

Reply:
Hey @javierjulio is this something you want me to do for this PR? Seems like they should be two separate changes?

Reply:
@javierjulio I've incorporated your suggestions.

Reply:
Thank you!